app.mjs

```diff
@@ -1,68 +1,76 @@
-import { fileURLToPath } from 'url'
+import createError from "http-errors"
+import express from "express"
+import path from "path"
+import { fileURLToPath } from "url"
 
 const __filename = fileURLToPath(import.meta.url)
 const __dirname = path.dirname(__filename)
 
-import cookieParser from 'cookie-parser'
-import dotenv from 'dotenv'
-import dotenvExpand from 'dotenv-expand'
+import cookieParser from "cookie-parser"
+import dotenv from "dotenv"
+import dotenvExpand from "dotenv-expand"
 
 let storedEnv = dotenv.config()
 dotenvExpand.expand(storedEnv)
 
-import logger from 'morgan'
-import cors from 'cors'
-import indexRouter from './index.mjs'
-import manifestRouter from './manifest/index.mjs'
-import projectRouter from './project/index.mjs'
-import pageRouter from './page/index.mjs'
-import lineRouter from './line/index.mjs'
+import logger from "morgan"
+import cors from "cors"
+import indexRouter from "./index.mjs"
+import manifestRouter from "./manifest/index.mjs"
+import projectRouter from "./project/index.mjs"
+import pageRouter from "./page/index.mjs"
+import lineRouter from "./line/index.mjs"
+import { jwtMiddleware, verifyWithAuth0 } from "./middlewares/verifyToken.mjs"
 
 let app = express()
 
 //Middleware to use
-app.use(logger('dev'))
+app.use(logger("dev"))
 app.use(express.json())
 app.use(express.urlencoded({ extended: true }))
 app.use(cookieParser())
-
 //Publicly available scripts, CSS, and HTML pages.
-app.use(express.static(path.join(__dirname, 'public')))
+app.use(express.static(path.join(__dirname, "public")))
 
 /**
  * For any request that comes through to the app, check whether or not we are in maintenance mode.
  * If we are, then respond with a 503 and a message.  Otherwise, continue on.
  */
-app.all('*', (req, res, next) => {
-  if (process.env.DOWN === 'true') {
-    res
-      .status(503)
-      .json({
-        message:
-          'TPEN3 services are down for updates or maintenance at this time.  We apologize for the inconvenience.  Try again later.',
-      })
+app.all("*", (req, res, next) => {
+  if (process.env.DOWN === "true") {
+    res.status(503).json({
+      message:
+        "TPEN3 services are down for updates or maintenance at this time.  We apologize for the inconvenience.  Try again later.",
+    })
   } else {
     next() //pass on to the next app.use
   }
 })
 
-app.use('/', indexRouter)
-app.use('/manifest', manifestRouter)
-app.use('/project', projectRouter)
-app.use('/line', lineRouter) 
-app.use('/page', pageRouter)
+//Check all request headers for authorization before calling any routes
+// app.use(jwtMiddleware()) // apply checks to all routes
+// app.use("/authenticated-route", jwtMiddleware()) // apply checks to specific routes
+
+const verifyToken = verifyWithAuth0()
+
+
+// app.use(verifyToken) //check all routes
+app.use("/page/*", verifyToken) //check specific routes
+
+app.use("/", indexRouter)
+app.use("/manifest", manifestRouter)
+app.use("/project", projectRouter)
```

Comment:
You can also be like `app.use("/project", verifyToken, projectRouter)` or for only some routes inside of projectRouter